proposals/description/tip_118.ts

```diff
@@ -0,0 +1,497 @@
+    {
+      target: 'volt',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.voltHoldingPCVDeposit, '10000000000000000000000000'],
+      description: 'Move 10M VOLT from the DAO timelock to the VOLT holding PCV deposit'
+    },
+
+    ///// Send gOHM to it's holding ERC20 deposit
+    {
+      target: 'gOHM',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.gOHMHoldingPCVDeposit, '577180000000000000000'],
+      description: 'Send 577 gOHM from the DAO timelock to the gOHM holding PCV deposit'
+    },
+
+    // Withdraw excess TRIBE from reward system
+    {
+      target: 'erc20Dripper',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '2508838570600494412126519'],
+      description: 'Withdraw 2.5M TRIBE from the ERC20 Dripper to the Core Treasury'
+    },
+    {
+      target: 'votiumBriber3Crvpool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '1725076846586787179639648'],
+      description: `
+      Withdraw all TRIBE from 3CRV Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 200k TRIBE pre-existing on this contract and then an additional 1.5M TRIBE 
+      that was harvested from the stakingTokenWrapperBribe3Crvpool.
+      `
+    },
+    {
+      target: 'votiumBriberD3pool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '232096077769383622085234'],
+      description: `
+      Withdraw all TRIBE from D3 Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 230k TRIBE that was harvested from the stakingTokenWrapperBribeD3pool.
+      `
+    },
+    {
+      target: 'rewardsDistributorAdmin',
+      values: '0',
+      method: '_grantComp(address,uint256)',
+      arguments: (addresses) => [addresses.core, '150000000000000000000000'],
+      description: `Withdraw excess 150k TRIBE from Rari delegator contract. There is 164k excess, leaving behind a buffer of 14k`
+    },
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'governorWithdrawTribe(uint256)',
+      arguments: (addresses) => ['26800000000000000000000000'],
+      description: `
+      Withdraw remaining TRIBE from the Tribal Chief to the Core Treasury, leaving behind enough to fund outstanding
+      rewards.
+      
+      Withdrawal amount = 
+      (Tribal Chief balance before harvest 
+            - (pending rewards, Uniswap-v2 FEI/TRIBE LP + Curve 3crv-FEI metapool LP + G-UNI DAI/FEI 0.05% fee tier)
+      
+      Withdrawal amount = 27.4M - 0.60M = 26.8M  
+      (There is ~564k in pending rewards, leaving behind an additional ~35k TRIBE as buffer)
```

Comment:
why do we leave a buffer of 35k ?

Reply:
Being conservative and making sure there is some excess in case of any weird issue around rounding, missed events etc. 

Feel like the cost of leaving behind some additional TRIBE is less than the potential cost of having to send more TRIBE there